collector/src/execute.rs

```diff
@@ -894,6 +900,44 @@ impl<'a> Processor for BenchProcessor<'a> {
     }
 }
 
+fn store_documentation_size_into_stats(stats: &mut Stats, doc_dir: &Path) {
+    match get_file_count_and_size(doc_dir) {
+        Ok((count, size)) => {
+            stats.insert("size:doc_files".to_string(), count as f64);
+            stats.insert("size:doc_bytes".to_string(), size as f64);
+        }
+        Err(error) => log::error!(
+            "Cannot get size of documentation directory {}: {:?}",
+            doc_dir.display(),
+            error
+        ),
+    }
+}
+
+/// Counts the number of files and the total size of all files within the given `path`.
+/// File size is counted as the actual size in bytes, i.e. the size returned by
+/// [std::path::Path::metadata].
+///
+/// Returns (file_count, size).
+pub fn get_file_count_and_size(path: &Path) -> std::io::Result<(u64, u64)> {
```

Comment:
This doesn't handle symlinks. Do `doc` directories contain symlinks? :thinking: @GuillaumeGomez

Reply:
It does for old macros but it's all I can think of.

Reply:
It's probably not super important, if there won't be a gazillion of them. They probably point to files within the doc folder anyway and symlinks don't take a lot of space.

Reply:
Yes, it's completely fine.